service-api/module/Application/config/module.config.php

```diff
@@ -237,7 +237,17 @@
                     'route'    => '/counter-service/:uuid/retrieve-letter',
                     'defaults' => [
                         'controller' => Controller\YotiController::class,
-                        'action'     => 'getPDFLetter',
+                        'action'     => 'retrievePDF',
+                    ],
+                ]
+            ],
+            'prepare_pdf_letter' => [
```

Comment:
Not sure the counter service internal API needs to map 1-2-1 to the Yoti API, part of the intent of having an internal api is to hide some of the yoti internals. You could wrap the prepare and retrieve into one `/letter` route, also means you avoid the verbs in urls problem..

Reply:
I think I would expect one API endpoint to handle:
- Create session
- Generate PDF
- Retrieve PDF and send to Sirius

The front-end shouldn't need to know what's going on behind-the-scenes, just telling the API "start the Yoti journey"

Reply:
Paper ID will need to call Sirius' API so send the PDF over, rather than Sirius calling Paper ID. As such that probably means the paper id frontend will call generate PDF then immediately call the Sirius documents API to store the letter against the case.